SAMtemplates/state_machines/GetMyPrescriptionsStateMachine.asl.json

```diff
@@ -60,7 +60,7 @@
         "FunctionName": "${GetStatusUpdatesFunctionArn}"
       },
       "InputPath": "$.statusUpdateData",
-      "Next": "Get Status Updates Result",
+      "Next": "Enrich Prescriptions",
```

Comment:
The catch block should also be updated to point to "Enrich Prescriptions".

Reply:
Sorted.